Windows-Installation_Instructions.txt

```diff
@@ -0,0 +1,51 @@
+INSTALLATION OF SIMPHONY, SIMPHONY-NCAD AND NFLUID
+==================================================
+
+1. Install Python 2.7
+   - https://www.python.org/downloads/
+     (Windows package python-2.7.11.msi)
+
+2. Configure Python to use MinGW
+   - create 'distutils.cfg' file under 'C:\Python27\Lib\distutils' folder with the text:
+[build]
+compiler = mingw32
```

Comment:
The Cython wiki discribes some of the reasons why mixing mingw and msvc in not a good idea
(https://github.com/cython/cython/wiki/CythonExtensionsOnWindows#less-useful-information)